### PR TITLE
schroot: add session examples, move --list to the top

### PR DESCRIPTION
**Pull request by ema**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/schroot.md
Comment (kbdharun) [inline pages/linux/schroot.md:26]:
```suggestion
- Begin a new session (A unique session ID is returned on standard output):
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/schroot.md
Comment (acuteenvy) [inline pages/linux/schroot.md:3]:
```suggestion
> Run a command or start an interactive shell with a different root directory. More customizable than `chroot`.
```

**Review by kbdharun — APPROVED** [review]

File: pages/linux/schroot.md
Comment (acuteenvy) [inline pages/linux/schroot.md:26]: #10436
```suggestion
- Begin a new session (a unique session ID is returned on `stdout`):
```

**Review by mfrw — APPROVED** [review]
LGTM once @acuteenvy suggestion is merged :)

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**
